### PR TITLE
New light and responsive design

### PR DESCRIPTION
**Pull request by rvennam**

### CODE REVIEW

**Comment by rvennam** [comment]
@opiethehokie please review http://rvennam-nodejs-cloudant.mybluemix.net/

`Squash and merge` please

File: app.js [hunk 1]
```diff
@@ -41,56 +41,42 @@ if ('development' == app.get('env')) {
...
-					if (err) { console.log('could not create db ', err); }
-				});
-				
-				db = cloudant.use(dbCredentials.dbName);
-				break;
+
+
 			}
 		}
 		if(db==null){
```
Comment (opiethehokie) [inline app.js]: @rvennam this warning should be removed now right?

Comment (rvennam) [reply app.js]: Yup, will fix.

**Comment by opiethehokie** [comment]
Also uploads aren't working for me. 

index.js:88 Uncaught TypeError: Cannot read property 'firstChild' of null

**Pull request merged**